### DISCUSSION TITLE
Poor usability: If there is an error when previewing scene, do NOT hide the error message behind a blank grey screen

### DISCUSSION THREAD

**Discussion by Johnrobmiller** [godotengine/godot-proposals#8674] [Editor]
This is a violation of the N/N usability heuristic number 1: Visibility of System Status (link: https://www.nngroup.com/articles/ten-usability-heuristics/)

When there is an error when previewing a scene, the user does not know that there is an error message awaiting them until they close the blank grey screen and go back to the editor. Furthermore, the user might not know or think to look there because they are not expecting an error message. Therefore, the visibility of the system status is needlessly obscure, which is poor usability.

Would it not be far better to display the error message somewhere on that blank grey screen?

**Comment by Calinou** [comment]
This is a bug: https://github.com/godotengine/godot/issues/78064
